src/components/app/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { h, Component } from 'preact';
+import { updater, toggle, When } from '../../lib/util';
+import Fab from '../fab';
+import Header from '../header';
+// import Drawer from 'async!../drawer';
+const Drawer = require('async!../drawer').default;
+import Home from '../home';
+import * as style from './style.scss';
+
+type Props = {
+	url?: String
+}
+
+type FileObj = {
+	id: any,
+	data: any,
+	error: Error | DOMError | String,
```

Comment:
I think `Error` is enough?

Reply:
Nah, it's a DOMError which is apparently not subclassed in the TS definition.